esp/bindings/http/platform/httpsecurecontext.cpp

```diff
@@ -22,6 +22,8 @@ class CHttpSecureContext : extends CEspBaseSecureContext
 {
 private:
     CHttpRequest* m_request;
+    StringAttr domainAuthData;
```

Comment:
It's been a long time since I paid close attention to the implementation. I forgot there was already a class between the interface and this class. The new data and methods should be lifted and shifted into the base.